Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,283 @@
+# Microsoft Mixed Reality Toolkit Release Notes
+
+- [Version 2.0.0](#version-200)
+
+## Version 2.0.0
+
+- [Upgrading projects](#upgrading-projects-to-200)
+- [What's new](#whats-new-in-200)
+- [Known issues](#known-issues-in-200)
+
+This release of the Microsoft Mixed Reality Toolkit 2.0.0 supports the following devices and platforms.
+
+- Microsoft HoloLens 2
+- Microsoft HoloLens (1st gen)
+- Windows Mixed Reality Immersive headsets
+- OpenVR
+
+The following software is required.
+
+- Microsoft Visual Studio (2017 or 2019) Community Edition or higher
+- Windows 10 SDK 18362 or later (installed by the Visual Studio Installer)
+- Unity 2018.4, 2019.1 or 2019.2
+
+### Upgrading projects to 2.0.0
```

Comment:
>Upgrading projects to 2.0.0 [](start = 4, length = 27)

It feels like this section should have a bold link to Updating.md

Otherwise, how will people learn about breaking changes?

Reply:
added.